libsplinter/src/store/mod.rs

```diff
@@ -91,6 +95,15 @@ pub fn create_store_factory(
                     "Failed to build connection pool".to_string(),
                 )
             })?;
+            let conn = pool
+                .get()
+                .map_err(|err| InternalError::from_source(Box::new(err)))?;
+            if !any_pending_postgres_migrations(&conn)? {
+                return Err(InternalError::with_message(String::from(
+                    "The splinter database is out of date. You must \
```

Comment:
If this test does fail, it implies that the database has not been initialized.  Same fix, but slightly different context.

Reply:
Updated message to say:
> This version of splinter requires migrations that are not yet applied to the database. Run `splinter database migrate` to apply migrations before running splinterd